src/elf/mod.rs

```diff
@@ -338,6 +344,44 @@ if_sylvan! {
             Ok((elf, src.len()))
         }
     }
+
+    fn gnu_hash_len(bytes: &[u8], offset: usize, ctx: Ctx) -> error::Result<usize> {
+        let buckets_num = bytes.pread_with::<u32>(offset, ctx.le)? as usize;
+        let min_chain = bytes.pread_with::<u32>(offset + 4, ctx.le)? as usize;
+        let bloom_size = bytes.pread_with::<u32>(offset + 8, ctx.le)? as usize;
```

Comment:
looking again, `gread_with` would  have been better here to automate the offset computations, but will update another time :)